site/downloads.markdown

```diff
@@ -27,7 +34,7 @@ The Haskell toolchain consists of the following tools:
 
 Alternatively, many operating systems provide GHC, cabal and Stack through their native package manager.  The packages are often out-of-date but if you prefer to use this method of installation then you will find useful links below.
 
-<p><a data-toggle="collapse" href="#collapse-linux" class="btn btn-xs btn-primary">Show Linux distros</a></p>
+<p><a data-toggle="collapse" href="#collapse-linux" class="btn btn-xs">Show Linux distros</a></p>
```

Comment:
Is this change (and the similar ones) necessary?

Reply:
OK, that makes sense! Can you please submit that as a separate PR? That one's easy to merge. No need to delay it by mixing it with a bigger one.

Reply:
In fact I made the PR myself: https://github.com/haskell-infra/www.haskell.org/pull/239

Could you copy across those screenshots to that PR?